domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Address.java

```diff
@@ -21,8 +23,8 @@
     @Size(max = 100, message = "City should not be longer than {max} characters")
     private final String city;
 
-    @NotBlank(message = "Postcode should not be empty")
-    @Size(max = 8, message = "Postcode should not be longer than {max} characters")
+    @NotNull
```

Comment:
any reason for the notblank to notnull change?

Reply:
As blank will be rejected by regex in `@Postcode`, so does not need it.